src/utils/firestoreUtils.ts

```diff
@@ -87,6 +87,27 @@ export function deleteBuilding(
   });
 }
 
+export function getNameFirestore(uid: string): string {
+  const docRef = firebase.firestore().collection("users").doc(uid);
+  let name = "";
+
+  docRef
+    .get()
+    .then((doc) => {
+      if (doc.exists) {
+        name = doc.data()?.name;
+      } else {
+        // doc.data() will be undefined in this case
+        console.log("No such document!");
```

Comment:
I like to have ids in logs

```suggestion
        console.log(`No user in "users" with uid ${uid}`);
```